ports/rp2/cyw43_configport.h

```diff
@@ -33,7 +33,12 @@
 #include "extmod/modnetwork.h"
 #include "pendsv.h"
 
+#if MICROPY_PY_BLUETOOTH_CYW43
+#define CYW43_CHIPSET_FIRMWARE_INCLUDE_FILE "wb43439A0_7_95_49_00_combined.h"
```

Comment:
This duplicates the definitions in the driver? I think we can remove the definition of CYW43_CHIPSET_FIRMWARE_INCLUDE_FILE from here

Reply:
Yes, remove it if possible.  But it may require changing the include directories to find `firmware/wb43439A0_7_95_49_00_combined.h`

Reply:
I removed it.  The include directory was already there.